src/app/shared/services/apis/gb3/gb3-print.service.ts

```diff
@@ -177,6 +180,35 @@ export class Gb3PrintService extends Gb3ApiService {
   }
 
   private createGb2WmsPrintItem(activeMapItem: ActiveMapItem, gb2WmsSettings: Gb2WmsSettings): PrintMapItem {
+    const customLayerParameters: {[index: string]: string} = {};
```

Comment:
Use an interface for this; it can be placed within `print.interface.ts`. Also instead of `index` I would call it `key`.